library/src/com/google/maps/android/geojson/GeoJsonPointStyle.java

```diff
@@ -0,0 +1,297 @@
+package com.google.maps.android.geojson;
+
+import com.google.android.gms.maps.model.BitmapDescriptor;
+import com.google.android.gms.maps.model.MarkerOptions;
+
+import java.util.Observable;
+
+/**
+ * A class that allows for GeoJsonPoint objects to be styled and for these styles to be translated
+ * into a MarkerOptions object
+ */
+public class GeoJsonPointStyle extends Observable implements GeoJsonStyle {
+
+    private final static String GEOMETRY_TYPE_REGEX = "Point|MultiPoint|GeometryCollection";
+
+    private final MarkerOptions mMarkerOptions;
+
+
+    /**
+     * Creates a new PointStyle object
+     */
+    public GeoJsonPointStyle() {
+        mMarkerOptions = new MarkerOptions();
+    }
+
+    /**
+     * Gets the type of geometries this style can be applied to
+     *
+     * @return type of geometries this style can be applied to
+     */
+    @Override
+    public String getGeometryType() {
+        return GEOMETRY_TYPE_REGEX;
+    }
+
+    /**
+     * Gets the alpha of the Point
+     *
+     * @return alpha of the Point
+     */
+    public float getAlpha() {
+        return mMarkerOptions.getAlpha();
+    }
+
+    /**
+     * Sets the alpha of the GeoJsonPoint
```

Comment:
What are the units here? 0..1 for opaque to transparent? -MAX_FLOAT to +MAX_FLOAT? -1 to 1?